### ISSUE TITLE
ImageView load event sometimes not fired

### ISSUE THREAD

**Issue by cpetrov** [eclipsesource/tabris-js#849]
Consecutively loading an image with the same source sometimes doesn't result into the 'load' event being fired. 

Working around this problem to show an activity indicator for the cases described below is very hard as it would require manually maintaining the loaded state for the image resource and configuration scenario.

|  | fired the very first time the image is loaded | fired on consecutive loading attempt when image object width and height are set | fired on consecutive loading attempt when image object width and height are not set |
| :-: | :-: | :-: | :-: |
| **Android** | ✓ | ✗ | ✗ |
| **iOS** | ✓ | **?** (fired for some values in the example below, e.g. w250, h250, but not for others, e.g. w220, h220) | ✓ |

Example: 

``` javascript
var page = new tabris.Page({topLevel: true});

var button = new tabris.Button({
  centerX: 0, top: 10,
  text: "Show me a llama"
}).appendTo(page);

button.on("select", function() {
  new tabris.ImageView({
    image: {
      src: "https://upload.wikimedia.org/wikipedia/commons/thumb/b/b9/Llama_lying_down.jpg/1920px-Llama_lying_down.jpg",
      // height: 200,
      // width: 200
    },
    centerX: 0, width: 200, height: 200, top: "prev() 10"
  }).on("load", function(widget, event) {
    console.log("Load event fired: " + JSON.stringify(event));
  }).appendTo(page);
});

page.open();
```

**Comment by mpost** [comment]
@cpetrov I would not expect a resize of the `ImageView` to trigger a `load` event. Would that be your expectation?

**Comment by mpost** [comment]
After clarifying the issue the following situation will be resolved Android client side for now: https://github.com/eclipsesource/tabris-js/issues/867#issuecomment-260642726

**Comment by mpost** [comment]
Fixed in the 2.0 stream.
